sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/GeneralScalarExpression.java

```diff
@@ -42,6 +42,30 @@
  *    <li>Since version: 3.3.0</li>
  *   </ul>
  *  </li>
+ *  <li>Name: <code>StartsWith</code>
```

Comment:
Can we move the doc to `Predicate` for these predicate expressions? We can link to it in this class
```
Please also see @{link Predicate} for the supported predicate expressions.
```